diktat-ruleset/src/main/kotlin/org/cqfn/diktat/api/DiktatCallback.kt

```diff
@@ -0,0 +1,9 @@
+package org.cqfn.diktat.api
+
+import java.util.function.BiConsumer
+
+/**
+ * Callback for diktat process
+ */
+@FunctionalInterface
+fun interface DiktatCallback : BiConsumer<DiktatError, Boolean>
```

Comment:
Up to you, but I'd use Kotlin-specific functional interfaces instead:

```suggestion
fun interface DiktatCallback : Function2<DiktatError, Boolean, Unit>
```

Either way, an instance can be created from a lambda:

```kotlin
val callback = DiktatCallback { error, booleanParam -> TODO() } 
```